crates/bevy_ui/src/accessibility.rs

```diff
@@ -147,7 +147,7 @@ pub(crate) struct AccessibilityPlugin;
 impl Plugin for AccessibilityPlugin {
     fn build(&self, app: &mut App) {
         app.add_systems(
-            Update,
+            PostUpdate,
             (calc_bounds, button_changed, image_changed, label_changed),
         );
     }
```

Comment:
`calc_bounds` looks like it needs to be run after transform propagation.

```suggestion
            PostUpdate,
            (
                calc_bounds.after(bevy_transform::TransformSystem::TransformPropagate), 
                button_changed, 
                image_changed, 
                label_changed
            ),
        );
    }
```